pkg/domain/infra/abi/play.go

```diff
@@ -290,7 +290,16 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		if v.Type == kube.KubeVolumeTypeConfigMap && !v.Optional {
 			vol, err := ic.Libpod.NewVolume(ctx, libpod.WithVolumeName(v.Source))
 			if err != nil {
-				return nil, errors.Wrapf(err, "cannot create a local volume for volume from configmap %q", v.Source)
+				if strings.Contains(err.Error(), "already exists") {
```

Comment:
please use proper error checking with `errors.Is()` and not string comparison.

Reply:
done.